dali/operators/reader/loader/lmdb.h

```diff
@@ -60,6 +60,8 @@ class IndexedLMDB {
     mdb_size_ = stat.ms_entries;
     LOG_LINE << "lmdb " << num_ << " " << db_path_
              << " has " << mdb_size_ << " entries" << std::endl;
+    MDB_val tmp_key, tmp_value;
+    mdb_cursor_get(mdb_cursor_, &tmp_key, &tmp_value, MDB_FIRST);
```

Comment:
Can you add the comment here why it is needed?

Reply:
Added, thank you